builder/googlecompute/step_create_image.go

```diff
@@ -46,6 +46,13 @@ func (s *StepCreateImage) Run(ctx context.Context, state multistep.StateBag) mul
 		}
 	}
 
+	if !config.AreLabelsValid() {
+		err := fmt.Errorf("Image: %s has invalid labels. The keys and values must start with a lowercase character, can only contain lowercase letters, numeric characters, underscores and dashes. They can be at most 63 characters long. International characters are allowed.", config.ImageName)
```

Comment:
This is good, but I wonder if we can be more directive regarding the kind of error message we return our users there. Error messages like those do point out the problem, but I think we can pinpoint which key/value is non-compliant there instead of returning this generic error.

If `configs.AreLabelsValid` returns an `[]error`, we can populate that collection with the exact key/value that is non-compliant, leading to easier troubleshooting.
What do you think?

Reply:
@lbajolet-hashicorp: Agreed that it would be a much better User Experience. I'll give it a go when time permits, unless you want to do it :)

Reply:
As you prefer, my goal was to release a new version of the plugin this week, so the earlier we can get this in, the better :)
If you think you'll have time to address that change, then I'll let you do this, but otherwise feel free to let me know, and I'll take a jab at that!

Reply:
Just gave it a go. Happy to get feedback

Reply:
Awesome, many thanks @DocEmmetBrown ! I'll review this ASAP